BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudCustomEntity.cs

```diff
@@ -467,6 +467,47 @@ public BrainCloudCustomEntity(BrainCloudClient client)
             _client.SendRequest(sc);
         }
 
+        /// <summary>
+        ///Gets a list of up to maxReturn randomly selected custom entities from the server 
+        ///based on the entity type and where condition.
+        /// </summary>
+        /// <remarks>
+        /// Service Name - CustomEntity
+        /// Service Operation - GetRandomEntitiesMatching
+        /// </remarks>
+        /// <param name="entityType">
+        /// </param>
+        /// <param name="whereJson">
+        /// </param>
```

Comment:
Sometimes, we dont have the description of these parameters, but in this case we do and it'd be good practice to tell our clients what these parameters are if we have the resources to it. 
check our api docs http://getbraincloud.com/apidocs/apiref/?csharp#capi-globalentity-getrandomentitiesmatching and try fill in what you can